source/beamline/AMPVControl.cpp

```diff
@@ -108,19 +123,23 @@ void AMReadOnlyPVControl::onReadPVInitialized() {
 	setUnits(readPV_->units());	// copy over the new unit string
 	setEnumStates(readPV_->enumStrings());
 	setDisplayPrecision(readPV_->displayPrecision());
-	setLowLimitValue(readPV_->lowerGraphicalLimit());
-	setHighLimitValue(readPV_->upperGraphicalLimit());
+
+	if (allowLowLimitValuePVUpdates_)
+		setLowLimitValue(readPV_->lowerGraphicalLimit());
+
+	if (allowHighLimitValuePVUpdates_)
+		setHighLimitValue(readPV_->upperGraphicalLimit());
 }
 
-void AMReadOnlyPVControl::onLowLimitPVInitialized()
+void AMReadOnlyPVControl::onLowLimitPVValueChanged()
 {
-	if (lowLimitPV_ && lowLimitPV_->isInitialized())
+	if (lowLimitPV_ && lowLimitPV_->isInitialized() && allowLowLimitValuePVUpdates_)
 		setLowLimitValue(lowLimitPV_->getDouble());
```

Comment:
Again, I wonder if these should check for the case that allowLowLimitValuePVUpdates_ is set to false (ie we've manually set the limits) but the incoming value had been set to a more constraining value?